fiftyone/utils/qwen3_8.py

```
@@ -0,0 +1,408 @@
+    Args:
+        config: a :class:`Qwen38ModelConfig`
+    """
+
+    def __init__(self, config):
+        self._processor = None
+        super().__init__(config)
+
+    def _download_model(self, config):
+        pass
+
+    def _load_model(self, config):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_load_model`

(ANN202)